src/omotes_sdk/omotes_interface.py

```diff
@@ -94,30 +94,36 @@ class OmotesInterface:
     """All available workflow types."""
     _workflow_config_received: threading.Event
     """Event triggered when workflow configuration is received."""
+    client_id: str
+    """Identifier of this SDK instance. Should be unique from other SDKs."""
 
-    def __init__(
-        self,
-        rabbitmq_config: RabbitMQConfig,
-    ):
+    def __init__(self, rabbitmq_config: RabbitMQConfig, client_id: str):
         """Create the OMOTES interface.
 
         NOTE: Needs to be started separately.
 
         :param rabbitmq_config: RabbitMQ configuration how to connect to OMOTES.
+        :param client_id: Identifier of this SDK instance. Should be unique from other SDKs.
         """
         self.broker_if = BrokerInterface(rabbitmq_config)
         self.workflow_type_manager = None
         self._workflow_config_received = threading.Event()
+        self.client_id = client_id
 
     def start(self) -> None:
         """Start any other interfaces and request available workflows."""
         self.broker_if.start()
+        self.broker_if.declare_exchange(OmotesQueueNames.omotes_exchange_name())
         self.connect_to_available_workflows_updates()
         self.request_available_workflows()
 
-        while not self._workflow_config_received.is_set():
-            logger.info("Waiting for workflow definitions to be received from the orchestrator...")
-            self._workflow_config_received.wait(timeout=5)
+        logger.info("Waiting for workflow definitions to be received from the orchestrator...")
+        if not self._workflow_config_received.wait(timeout=60):
```

Comment:
Do we want to set this as a configurable variable?

Reply:
Good one! Fixed: https://github.com/Project-OMOTES/omotes-sdk-python/pull/53/commits/bbea78f85d9095e092abe087f041f827831637fe